admin-dev/themes/new-theme/package.json

```diff
@@ -57,33 +57,35 @@
     "babel-register": "^6.26.0",
     "bootstrap": "~4.0.0",
     "chai": "^4.2.0",
-    "clean-webpack-plugin": "^0.1.19",
-    "copy-webpack-plugin": "^4.6.0",
-    "css-loader": "^0.27",
-    "eslint": "^6.7.2",
+    "clean-webpack-plugin": "^3.0.0",
+    "copy-webpack-plugin": "^5.1.1",
+    "css-loader": "^3.4",
+    "eslint": "^6.8.0",
     "eslint-config-airbnb-base": "^14.0.0",
     "eslint-config-prestashop": "0.0.2",
-    "eslint-import-resolver-webpack": "^0.11.1",
+    "eslint-import-resolver-webpack": "^0.12.1",
     "eslint-plugin-html": "^6.0.0",
-    "eslint-plugin-import": "^2.18.2",
+    "eslint-plugin-import": "^2.20.0",
     "eslint-plugin-vue": "^6.0.1",
-    "exports-loader": "^0.6.3",
+    "exports-loader": "^0.7.0",
     "expose-loader": "^0.7.5",
-    "extract-text-webpack-plugin": "^2.1.2",
-    "file-loader": "^0.10",
+    "file-loader": "^5.0",
     "imports-loader": "^0.8.0",
     "loader-utils": "^1.2.3",
     "mocha": "^6",
-    "node-sass": "^4.12.0",
+    "mini-css-extract-plugin": "^0.8.0",
+    "node-sass": "^4.13.1",
     "path": "^0.12.7",
     "postcss-loader": "^3.0.0",
-    "sass-loader": "^6.0.7",
-    "style-loader": "^0.23.0",
+    "sass-loader": "^8.0.2",
+    "style-loader": "^1.1.3",
     "uglify-save-license": "^0.4.1",
-    "uglifyjs-webpack-plugin": "^1.3.0",
-    "vue-loader": "^11",
+    "uglifyjs-webpack-plugin": "^2.2.0",
+    "vue-loader": "^15",
     "vue-style-loader": "^4.1.2",
-    "webpack": "^2.7"
+    "webpack": "^4.41",
+    "webpack-cli": "^3.3.10",
```

Comment:
@NeOMakinG Have you checked license of `webpack-cli` & `webpack-fix-style-only-entries` ?

Reply:
MIT and ISC @Progi1984

Reply:
Is it ok for you @ttoine ?

Reply:
I confirm that MIT and ISC are permissive licences compatible with the PrestaShop project's OSL and AFL license.
May I ask if this is for the core, or for the Classic theme?

Reply:
Core new-theme